CHANGELOG.md

```diff
@@ -7,12 +7,15 @@ pull request if there was one.
 
 Current
 -------
-
 ### Added:
+- [Refactor DatasourceMetadataService to fit composite table needs](https://github.com/yahoo/fili/pull/173)
```

Comment:
Refactor is just the PR name, the actual content is what matters. I would say refactor should be things that maintain the same functionality but just changing how to achieve the same functionality. I put this into added because it adds additional capability to `DataSourceMetadataService`.